### ISSUE TITLE
Bad performance on EK100 MIR FT task

### ISSUE THREAD

**Issue by dhimitriosduka1** [zhaoyue-zephyrus/AVION#14]
Dear Yue,

Thanks for this amazing repo. I'm trying to fine-tune on EK100 for the MIR task. However, the results that I'm getting are very different from yours. There's a difference of ~4% in the average mAP when using the ViT base model.
Looking at the training metrics, I feel there could be something wrong with the default hyperparameter values. The loss seems to saturate very fast. I've attached the training loss of one of my runs. 

![image](https://github.com/user-attachments/assets/ccb3da84-395b-45da-ad60-38ca5b32112f)

If possible, can you provide the logs of the EK100 MIR fine-tuning so that I can take a look at them?

Thanks in advance,
Dhimitrios

**Comment by zhaoyue-zephyrus** [comment]
Hi @dhimitriosduka1 

Thanks for trying out our code. Could you tell me your configuration, particularly per-device batch size and the number of GPUs being used? The total batch size might have some effect on the performance due to the training loss that the MIR task relies on.

Per your request, I also upload the training log below.

[log.txt](https://github.com/user-attachments/files/17962960/log.txt)
